docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use an official Python runtime as a parent image
-FROM osrf/ros:noetic-desktop-full
+FROM osrf/ros:foxy-desktop
```

Comment:
Might make sense to use [ros:foxy-ros1-bridge-focal](https://github.com/osrf/docker_images/blob/f7d378b9d6ed10c72a2f686554f7b712e921c1aa/ros/foxy/ubuntu/focal/ros1-bridge/Dockerfile) so that both versions are available.